graphql/tests/host/queries/hostMetadata-no-running-task.graphql

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
each resolver typically has it's own test directory, but i think it's ok that this one shares a test directory with the host resolver. in future though they should all be separated